[FCT-980] App kit background changes (#3596)
commercetools/merchant-center-application-kit: .changeset/shy-pillows-confess.md

---
'@commercetools-frontend/application-components': minor
'@commercetools-frontend/assets': minor
---

> Added a gradient background and smaller commercetools logo behind the workspaces UI feature flag. These changes are for the public pages in the MC Auth app, but they use the public-page-layout component in app-kit.

Reviewer: ```suggestion
Added a gradient background and smaller commercetools logo to the public-page-layout component as part of an ongoing feature effort
```
